### PR TITLE
feat(Popup): add `positionFixed` to support fixed mode in `Popper.JS`

### LINKED ISSUE #3718: Popup: wrong positioning when `mountNode` uses `position: fixed`
### Steps
-add Popup control
-set mountNode property

### Expected Result
Popup should be renderer in a set node and visually be placed there

### Actual Result
Popup is rendered in a set node but visually is NOT placed there
It is placed far away outside
Wrong calculation of `transform` css value

### Version
0.87.3

In the version 0.86.0 all is ok

### PR DESCRIPTION
**Pull request by fxOne**
Fixes #3718.

Add support for popper.js to position the popup correctly if used with a mountNode inside a `position: fixed` container

### CODE REVIEW

File: src/modules/Popup/Popup.js [hunk 1]
```diff
@@ -153,6 +156,7 @@ export default class Popup extends Component {
     on: ['click', 'hover'],
     pinned: false,
     position: 'top left',
+    positionFixed: false,
```
Comment (layershifter) [inline src/modules/Popup/Popup.js]:
```suggestion
```

It's not required to set defaults to `false`

File: test/specs/modules/Popup/Popup-test.js [hunk 2]
```diff
@@ -289,6 +289,20 @@ describe('Popup', () => {
     })
   })
 
+  describe('positionFixed', () => {
+    it(`is "false" by default`, () => {
+      wrapperMount(<Popup open />)
+
+      wrapper.should.have.prop('positionFixed', false)
```
Comment (layershifter) [inline test/specs/modules/Popup/Popup-test.js]:
```suggestion
      wrapper.should.not.have.prop('positionFixed')
```

https://github.com/Semantic-Org/Semantic-UI-React/pull/3760/files#r317650640

**Review by layershifter — APPROVED** [review]
Left minor comments, looks good to me ❤️

**Comment by fxOne** [comment]
Thanks for the feedback, I removed the unnecessary code.

File: src/modules/Popup/Popup.d.ts [hunk 3]
```diff
@@ -107,6 +107,9 @@ export interface StrictPopupProps extends StrictPortalProps {
     | 'top center'
     | 'bottom center'
 
+  /** Enables the popper position 'fixed' mode */
```
Comment (layershifter) [inline src/modules/Popup/Popup.d.ts]:
```suggestion
  /** Tells `Popper.js` to use the `position: fixed` strategy to position the popover. */
```

File: src/modules/Popup/Popup.js [hunk 4]
```diff
@@ -127,6 +127,9 @@ export default class Popup extends Component {
     /** Position for the popover. */
     position: PropTypes.oneOf(positions),
 
+    /** Enables the popper position 'fixed' mode */
```
Comment (layershifter) [inline src/modules/Popup/Popup.js]:
```suggestion
    /** Tells `Popper.js` to use the `position: fixed` strategy to position the popover. */
```

**Pull request merged**